src/Registry.php

```diff
@@ -80,6 +80,67 @@ public function __clone()
 		$this->data = unserialize(serialize($this->data));
 	}
 
+	/**
+	 * Set a registry value.
+	 * Example: $registry->anyProperty = 'Value'; as $registry->set('anyProperty', 'Value');
+	 *
+	 * @param   string  $path   Registry Path (e.g. joomla.content.showauthor)
+	 * @param   mixed   $value  Value of entry
+	 *
+	 * @return  mixed  The value of the that has been set.
+	 *
+	 * @since   1.0
+	 */
+	public function __set(string $path = '', $value = null): void
```

Comment:
Having a default string will allow this to be called without a path which kinda defeats the object of the method right?